generators/app/templates/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "suitcss-<%= modulePackageName %>",
```

Comment:
This seems to output `component` instead of `components`, like:

`suitcss-component-test`